packages/govuk-frontend/src/govuk/i18n.mjs

```diff
@@ -70,6 +72,7 @@ export class I18n {
    * Takes a translation string with placeholders, and replaces the placeholders
    * with the provided data
    *
+   * @internal
```

Comment:
**question** I think outside of `t`, `I18n`'s methods are `@private` (to the class) rather than `@internal` (to our package). Does flagging them as internal help with something else (TypeScript maybe)?

Reply:
No not really, we'd just need to watch out for these ones:

1. `getPluralSuffix()`
1. `getPluralRulesForLocale()`
1. `selectPluralFormUsingFallbackRules()`

We call them externally from tests so it's wrong to make them `@private`

Reply:
Is that something that TypeScript would flag? In which case we can go with the process of trying to lock things as `@private`, if TypeScript flags (for tests, for ex) `@internal`.

Reply:
Sadly we also add mocks/spies to private methods:

```mjs
jest.spyOn(i18n, 'hasIntlPluralRulesSupport')
  .mockImplementation(() => false)
```

I'll keep them all as `@internal` and we can assess how we test "non-public" class interfaces another day!

Reply:
Yeah, may be more efficient to go for `@internal` all the way + some convention (the commonplace leading `_`) for things that are meant to only be accessed from within the class. I've scheduled something in dev catch up to discuss the pains of marking things properly `@private` so we can chose knowingly whether to embrace it.